tests/core/recipes/orca_recipes/test_orca_recipes.py

```diff
@@ -141,3 +155,68 @@ def test_ase_relax_job_store(tmp_path, monkeypatch):
         assert "orca.xyz.gz" in os.listdir(Path(output["dir_name"], f"step{i}"))
     assert len(output["steps"]) == nsteps
     assert "attributes" in output["steps"][0]
+
+
+@pytest.mark.skipif(os.name == "nt", reason="mpirun not available on Windows")
+def test_freq_job(tmp_path, monkeypatch):
+    monkeypatch.chdir(tmp_path)
+
+    atoms = molecule("H2")
+
+    output = freq_job(
+        atoms,
+        xc="hf",
+        basis="def2-svp",
+        charge=0,
+        spin_multiplicity=1,
+        nprocs=2,
```

Comment:
Would you mind removing `nprocs=2` so we can run this on Windows too (i.e. you can remove the decorator afterwards)? Same as below. I don't think `nprocs` is needed for this test.